lightspeed/flame
# CRS-651: Build icons and flags on pre-commit hook when svgs are changed

## Description

Run pre-commit hook to build icons and flags when either have changed.

## How to test?

- Make a change to any SVG in either the flags or icons directories
- Attempt to commit
- Notice that the commit has new build files as well

## Checklist
